google/cloud/forseti/common/gcp_api/api_data_helpers.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for data transformation."""
+
+from google.cloud.forseti.common.gcp_type.resource_util import create_resource
+
+
+def extract_project_ids(project_response):
+    """Extract a list of project ID's from the CRM project response.
+
+    Args:
+        project_response (list): The response from
+            CloudResourceManagerApiClient.get_projects()
+
+    Returns:
+        list: A list of project ID's
+
+    """
+    project_id_result = []
+
+    for projects in project_response:
+        for project in projects['projects']:
+            project_id_result.append(project['projectId'])
+    return project_id_result
+
+
+def cast_to_gcp_resource(resource):
```

Comment:
This needs to be a helper method under google/cloud/forseti/common/gcp_type/resource_util.py, same below. They shouldn't be added to a new module under gcp_api because they deal specifically with creating gcp_type objects.